py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -1,10 +1,17 @@
 import numpy as np
 import matplotlib.pyplot as plt
+import os
 from typing import Union, Optional
 
 from ...io.datastructure import PointList, PointListArray
 from ..utils import tqdmnd, electron_wavelength_angstrom
+from .utils import Orientation, OrientationMap, axisEqual3D
 
+try:
+    from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+    from pymatgen.core.structure import Structure
+except ImportError:
+    pass
```

Comment:
This will fail silently, should we raise a warning about some features not being available?

Reply:
Yes we should drop a warning - I think it's this way so you can still run everything without pymatgen.

@bsavitzky we could just add pymatgen as a dependency, it seems quite lightweight . . .